### PR TITLE
Remove epam/pipeline-builder from npm dependencies

### PR DESCRIPTION
**Pull request by jmthibault79**
- [ ] **Submitter**: Include the JIRA issue number in the PR description
- [ ] **Submitter**: Check documentation and code comments. Add explanatory PR comments if helpful.
- [ ] **Submitter**: If you changed a URL that is used elsewhere (e.g. in an email), comment about where it is used and ensure the dependent code is updated.
- [ ] **Submitter**: JIRA ticket checks:
  * Acceptance criteria exists and is met
  * Note any changes to implementation from the description
  * To Demo flag is set
  * Release Summary is filled out, if applicable
  * Add notes on how to QA
- [ ] **Submitter**: Update RC_XXX release ticket with any config or environment changes necessary
- [ ] **Submitter**: Update FISMA documentation if changes to:
  * Authentication
  * Authorization
  * Encryption
  * Audit trails
- [ ] **Submitter**: If you're adding new libraries, sign us up to security updates for them
- [ ] **Submitter**: If you're adding new automated UI tests, review the test plan with QA
* Review cycle:
  * LR reviews
  * Rest of team may comment on PR at will
  * **LR assigns to submitter** for feedback fixes
  * Submitter rebases to develop again if necessary
  * Submitter makes further commits. DO NOT SQUASH
  * Submitter updates documentation as needed
  * Submitter **reassigns to LR** for further feedback
- [ ] **TL** sign off
- [ ] **LR** sign off
- [ ] **Product Owner** sign off
- [ ] **Submitter**: Verify all tests go green, including CI tests and automated UI tests.
- [ ] **Submitter**: Squash commits and merge to develop. If adding test code, merge application code and test code at the same time.
- [ ] **Submitter**: Delete branch after merge
- [ ] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [ ] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

**Comment by jmthibault79** [comment]
Removes a transitive dependency vulnerability.

**Comment by jmthibault79** [comment]
fiab-start failed with Sam 503.  jenkins retest

File: package-lock.json [hunk 1]
```diff
@@ -1020,7 +1020,8 @@
     "core-util-is": {
```
Comment (ghost) [inline package-lock.json:1020]: just a general question: did you make changes to this file manually, or was this the result of running `npm <command>`? if the latter, which command did you use?

**Review by ghost — APPROVED** [review]
one question inline, otherwise lgtm 👍

File: package-lock.json (hunk 1 above)
Comment (zarsky-broad) [reply package-lock.json:1020]: `npm rm pipeline-builder`, I'd wager

**Review by zarsky-broad — APPROVED** [review]
Trailing commas are valid in js, so you don't actually even have to change the markdown dep line.

File: package-lock.json (hunk 1 above)
Comment (zarsky-broad) [reply package-lock.json:1020]: (lockfiles should never be modified manually)

Comment (jmthibault79) [reply package-lock.json:1020]: npm rm pipeline-builder

**Comment by jmthibault79** [comment]
auto-tests: AuthDomainOwnerSpec has a 504 RestTimeout.  I don't see how that can be related.

**Comment by davidangb** [comment]
Could you please add code comments to `wdl.cljs`, `pipeline_builder.cljs`, and maybe `pipeline-builder.scss` (and anywhere else I missed) to denote that re-enabling the pipeline builder will require updates to dependencies?

**Comment by jmthibault79** [comment]
jenkins retest

**Comment by jmthibault79** [comment]
https://github.com/DataBiosphere/firecloud-app/issues/150

**Review by davidangb — APPROVED** [review]
:+1: did not test - thanks for this PR!

**Comment by jmthibault79** [comment]
fiab-start failed rendering configs.

jenkins retest

**Comment by jmthibault79** [comment]
systemic test env issue caused global failures.  discussed and fixed.

jenkins retest

**Pull request merged**